### PR TITLE
refactor: more concise warning message

### PR DESCRIPTION
**Pull request by gahtan-syarif**

### CODE REVIEW

**Comment by Disservin** [comment]
this loses information, about it being the last one?

**Comment by gahtan-syarif** [comment]
> this loses information, about it being the last one?

Updated, also removed the score message since the conditional does not check for score

**Comment by Disservin** [comment]
> since the conditional does not check for score

but it does... https://github.com/Disservin/fastchess/blob/f8da6c3161aa9d9fa74cbead89281ad3497165c9/app/src/engine/uci_engine.cpp#L296

**Comment by gahtan-syarif** [comment]
ahh didnt see that

**Comment by gahtan-syarif** [comment]
updated

File: app/src/engine/uci_engine.cpp [hunk 1]
```diff
@@ -329,8 +329,7 @@ std::vector<std::string> UciEngine::lastInfo() const {
 
     if (last_info.empty()) {
         Logger::warn<true>(
-            "Warning; No info line found in the last line which includes the "
-            "score from {}",
+            "Warning; Last info string with score (cp) not found from {}",
```
Comment (Disservin) [inline app/src/engine/uci_engine.cpp:332]: why `(cp)` .. this is wrong `score mate` or `score cp`

Comment (gahtan-syarif) [reply app/src/engine/uci_engine.cpp]: right, updated

Comment (Disservin) [reply app/src/engine/uci_engine.cpp]: okay but why did you know add eval? there is no eval in the last info string line.. we expect score not eval

Comment (gahtan-syarif) [reply app/src/engine/uci_engine.cpp]: oh, i thought theyre the same thing(?) but removed it anyways

**Pull request merged**